src/libraries/testPackages/testPackages.proj

```diff
@@ -29,6 +29,8 @@
 
     <!-- runtime.native.System.IO.Ports requires all runtime legs to be built in order to test -->
     <ExcludePackages Include="runtime.native.System.IO.Ports" />
+    <!-- Microsoft.NET.WebAssembly.Threading only has ref assemblies, no runtime items -->
+    <ExcludePackages Include="Microsoft.NET.WebAssembly.Threading" />
```

Comment:
@ericstj Should ref only packages be excluded from package testing?

Reply:
The package testing actually complained "you don't have any runtime items in this package" so it would be nice to have a way to say "ensure that I don't have any runtime items in this package":
```
D:\a\_work\1\s\artifacts\bin\testPackages\packageTest.targets(94,5): error : Could not locate any runtime items from Package Microsoft.NET.WebAssembly.Threading [D:\a\_work\1\s\artifacts\bin\testPackages\projects\Microsoft.NET.WebAssembly.Threading\project.csproj]
```

Reply:
Presumably by setting `SkipVerifyRuntime` to true in a file under https://github.com/dotnet/runtime/tree/main/src/libraries/testPackages/packageSettings.

Reply:
> @ericstj Should ref only packages be excluded from package testing

Yeah -- I wouldn't have expected we'd have many `ref` only packages though we've seemed to found new ways to use them to do interesting experiments.  If that continues maybe we should simplify the pattern.